src/storage-adapters/git/index.test.js

```diff
@@ -45,7 +45,6 @@ describe('GitAdapter', () => {
     subject = new GitAdapter({
```

Comment:
subject has been a bit confusing for me to understand.

What about `adapter`?

Reply:
It's a common practice used through the whole codebase so I won't modify it here but we could discuss about this if we want to change this globally.